ledger/sandbox/src/main/scala/com/digitalasset/platform/store/ActiveLedgerStateManager.scala

```diff
@@ -112,7 +105,7 @@ class ActiveLedgerStateManager[ALS <: ActiveLedgerState[ALS]](initialState: => A
                   None
                 case None =>
                   // Contract not known
-                  Some(InactiveDependencyError(cid, predType))
+                  Some(Inconsistent(s"Could not lookup contract $cid"))
```

Comment:
Is this an improvement related to the fix? I lack bg. on the previous error reporting but can you elaborate here?

Reply:
This is needed, because I had to fix the test to expect the correct error message due to the validation error. Without this change, the tests for sandbox-classic would not have passed, because sandbox-classic would produce a different error message.
So this is basically making kvutils based ledgers and sandbox-classic more consistent.